src/pngwriter.cc

```diff
@@ -787,23 +795,24 @@ int pngwriter::read(int x, int y, int colour) const
 
 	if(bit_depth_ == 8)
 	  {
+	     int const scale8To16Bit = 257;   // x/255*65535 = x*257
```

Comment:
this comment is not clear to me. integer `255*65535` is `16711425` and `1/16711425` is `0`. not `*257`